src/main/preload.ts

```diff
@@ -15,6 +15,9 @@ export type Channels =
   | 'versions:list'
   | 'window:close'
   | 'window:minimize'
+  | 'config:read'
+  | 'config:write'
+  | 'user:write'
```

Comment:
The channel order is broken. Notice that I order them alphabetically (if it doesn't give me OCD/TOC xD)